klippa/src/lib.rs

```diff
@@ -37,6 +39,91 @@ const MAX_NESTING_LEVEL: u8 = 64;
 // See <https://github.com/googlefonts/fontations/issues/997>
 const MAX_GID: GlyphId = GlyphId::new(0xFFFFFF);
 
+#[derive(Clone, Copy, Debug)]
+pub struct SubsetFlags(u16);
+
+impl SubsetFlags {
+    //all flags at their default value of false.
+    pub const SUBSET_FLAGS_DEFAULT: Self = Self(0x0000);
+
+    //If set hinting instructions will be dropped in the produced subset.
+    //Otherwise hinting instructions will be retained.
+    pub const SUBSET_FLAGS_NO_HINTING: Self = Self(0x0001);
+
+    //If set glyph indices will not be modified in the produced subset.
+    //If glyphs are dropped their indices will be retained as an empty glyph.
+    pub const SUBSET_FLAGS_RETAIN_GIDS: Self = Self(0x0002);
+
+    //If set and subsetting a CFF font the subsetter will attempt to remove subroutines from the CFF glyphs.
```

Comment:
Can you add a note to all of the unimplemented flags that that are not implemented yet.